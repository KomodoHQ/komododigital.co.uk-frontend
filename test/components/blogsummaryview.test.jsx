import React from 'react';
import BlogSummaryView from '../../src/components/blogsummaryview.tsx';
import renderer from 'react-test-renderer';

test('BlogSummaryView element is rendered', () => {
  const component = renderer.create(<BlogSummaryView />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
