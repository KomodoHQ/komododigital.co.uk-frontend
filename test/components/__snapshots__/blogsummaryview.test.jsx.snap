// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogSummaryView element is rendered 1`] = `
<div>
  <div>
    <img
      height="150"
      src="https://www.komododigital.co.uk/static/media/logo.50a4b652.svg"
      width="300"
    />
  </div>
  <h1 />
  <span>
    22 September 2018
  </span>
</div>
`;
